From: Ben Klopfenstein <benklop@gmail.com>
Subject: [PATCH] Install generated metainfo from the build directory

com.dosbox_x.DOSBox-X.metainfo.xml is generated by configure from its .in
template, so it lands in the build directory. Installing it from $(srcdir)
fails whenever the build is out of tree.

Upstream-Status: Pending

--- a/Makefile.am
+++ b/Makefile.am
@@ -160,7 +160,7 @@
 	mkdir -p $(DESTDIR)$(prefix)/share/applications
 	install -m 644 $(srcdir)/contrib/linux/com.dosbox_x.DOSBox-X.desktop $(DESTDIR)$(prefix)/share/applications
 	mkdir -p $(DESTDIR)$(prefix)/share/metainfo
-	install -m 644 $(srcdir)/contrib/linux/com.dosbox_x.DOSBox-X.metainfo.xml $(DESTDIR)$(prefix)/share/metainfo
+	install -m 644 $(builddir)/contrib/linux/com.dosbox_x.DOSBox-X.metainfo.xml $(DESTDIR)$(prefix)/share/metainfo
 	mkdir -p $(DESTDIR)$(mandir)/man1
 	install -m 644 $(srcdir)/contrib/linux/dosbox-x.1 $(DESTDIR)$(mandir)/man1
 	mkdir -p $(DESTDIR)$(prefix)/share/bash-completion/completions
